.github/workflows/test.yaml

```
@@ -5,16 +5,102 @@ on:
+        run: |
+          if [[ "${{ matrix.tests.type }}" == "Integration Tests - Part"* ]]; then
+            npm run test:ember -- --file-path=/irene/tests/integration/!(components/security)/**/*.js --split=7 --partition=${{ matrix.partition }} --parallel=7 --load-balance --path=dist --wmmf
+          else
+            npm run test:ember -- ${{ matrix.tests.commandArgs }} --path=dist --wmmf
+          fi
 
-      - name: Run tests
-        run: npm test
+      - name: Job Summary
+        if: always()
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)